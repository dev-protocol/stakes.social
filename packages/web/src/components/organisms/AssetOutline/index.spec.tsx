import React from 'react'
import { render } from '@testing-library/react'
import { AssetOutline } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/devprtcl/hooks')

describe(`${AssetOutline.name}`, () => {
  test('Snapshot', () => {
    const component = render(<AssetOutline propertyAddress="property-address" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
