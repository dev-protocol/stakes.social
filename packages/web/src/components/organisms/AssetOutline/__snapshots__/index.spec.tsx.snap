// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetOutline Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-gsDKAQ sc-dkPtRN dkxVID fEdfFf"
    >
      <div
        class="sc-jRQBWg brOWgw"
      >
        <h2>
          Author
        </h2>
        <div
          class="sc-iCfMLu dsWAqZ"
        >
          <span>
            property_name
          </span>
          <div
            class="sc-gKclnd TRfYI"
          >
            40124
          </div>
        </div>
        <div
          class="sc-pVTFL eJAwIh"
        >
          <span
            class="sc-furwcr djgpqw"
          >
            <a
              class="sc-jrQzAO fwHvGS"
              href="https://etherscan.io/address/hex_address"
              rel="noreferrer"
              target="_blank"
            >
              hex_address
            </a>
          </span>
          <span
            aria-label="export"
            class="anticon anticon-export"
            color="grey"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="export"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M888.3 757.4h-53.8c-4.2 0-7.7 3.5-7.7 7.7v61.8H197.1V197.1h629.8v61.8c0 4.2 3.5 7.7 7.7 7.7h53.8c4.2 0 7.7-3.4 7.7-7.7V158.7c0-17-13.7-30.7-30.7-30.7H158.7c-17 0-30.7 13.7-30.7 30.7v706.6c0 17 13.7 30.7 30.7 30.7h706.6c17 0 30.7-13.7 30.7-30.7V765.1c0-4.3-3.5-7.7-7.7-7.7zm18.6-251.7L765 393.7c-5.3-4.2-13-.4-13 6.3v76H438c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h314v76c0 6.7 7.8 10.5 13 6.3l141.9-112a8 8 0 000-12.6z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div>
        <p>
          Staking Ratio
        </p>
        <div
          class="sc-gsDKAQ sc-hKwDye dkxVID bTfJjR"
        >
          <div
            class="sc-bdvvtL hGdENr"
          />
          <div>
            <span
              class="sc-eCImPb hrFZps"
            >
              26
              %
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
